src/PKSim.Core/Services/DistributedParameterToTableParameterConverter.cs

```diff
@@ -275,20 +290,61 @@ private bool parameterShouldBeDefinedAsTable(IDistributedParameter parameter)
          return !parameter.NameIsOneOf(CoreConstants.Parameters.MEAN_HEIGHT, CoreConstants.Parameters.MEAN_WEIGHT);
       }
 
-      private void createPopulationTableParameter(KeyValuePair<string, IDistributedParameter> individualParameter, PopulationSimulation populationSimulation,
-         IEnumerable<ParameterDistributionMetaData> allDistributionsForMaleParameter, IEnumerable<ParameterDistributionMetaData> allDistributionsForFemaleParameter)
+      private void createPopulationTableParameter(
+         string individualParameterPath,
+         IDistributedParameter individualParameter,
+         PopulationSimulation populationSimulation,
+         IReadOnlyList<ParameterDistributionMetaData> allDistributionsForMaleParameter,
+         IReadOnlyList<ParameterDistributionMetaData> allDistributionsForFemaleParameter)
       {
-         if (populationSimulation == null) return;
-         var parameter = individualParameter.Value;
+         if (populationSimulation == null)
+            return;
 
-         addAgingDataToPopulationSimulation(populationSimulation, individualParameter.Key, parameter,
+         addAgingDataToPopulationSimulation(populationSimulation, individualParameterPath, individualParameter,
             p =>
             {
-               var distributions = allDistributionsWithAgeStrictBiggerThanOriginData(allDistributionsForMaleParameter, allDistributionsForFemaleParameter, p.OriginData).ToList();
+               var distributions = allDistributionsWithAgeStrictBiggerThanOriginData(allDistributionsForMaleParameter, allDistributionsForFemaleParameter, p.OriginData);
                return createTableFormulaFrom(p, distributions);
             });
       }
 
+      private void createPopulationHeightTableParameter(
```

Comment:
I could not use the deafult method because the height parmaeter is not distributed. I could have messed around with the original function but its' complicated enough already so I separated the tasks